src/styles/variables.scss

```diff
@@ -0,0 +1,22 @@
+$font-style: Roboto;
+
+$small-font: 14px;
```

Comment:
sizes for fonts and icons should be in EM/REM

Reply:
done